### PR TITLE
feat(skia-draw): add basic text support

### PR DESCRIPTION
**Pull request by HaakonSvane**
- Refactor Canvas control mechanism
  - Create `useCanvasControl` hook
  - Create `CanvasControlProvider`
  - All canvas state is now collected in the context, accessible and modifiable through the hook
- Add support for adding text to the canvas
- Add text controls to `EDSControlPanel`
- Update documentation 
- Upgrade Chronicles to expo 49.0.21
- Upgrade peer dependency version on `@shopify/react-native-skia` to 0.1.225 (needed for text support)

### CODE REVIEW

**Comment by HaakonSvane** [comment]
Screenshot 2023-12-13 at 07 06 35

**Review by tormodAase — APPROVED** [review]
Looks good! I think the changset should have way more information, otherwise I have nothing to add

**Review by tormodAase — DISMISSED** [review]
Looks good! I think the changset should have way more information, otherwise I have nothing to add

File: .changeset/eighty-ligers-tap.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+"@equinor/react-native-skia-draw": minor
+---
+
+**Breaking change**: `SnapShotHandle` type is now replaced with `CanvasImageControls` type.
```
[the remark this answers is not in the document]
Comment (HaakonSvane) [reply .changeset/eighty-ligers-tap.md:5]: Added a new changeset for this as well!

**Review by tormodAase — DISMISSED** [review]

**Review by tormodAase — APPROVED** [review]

**Pull request merged**